CHANGELOG.md

```diff
@@ -103,6 +103,7 @@ BUG FIXES
   -  [x/stake] \#2021 Fixed repeated CLI commands in staking
 
 * Gaia
+  - [x/stake] [#2077](https://github.com/cosmos/cosmos-sdk/pull/2077) Fixed invalid cliff power comparison
```

Comment:
We shouldn't need to put the links in the pending file I don't think

Reply:
Ok -- I'll stop doing that.